test/test.js

```diff
@@ -458,6 +458,19 @@ function checkRefTestResults(browser, id, results) {
   });
 }
 
+function browserTestReportHandler(req,res) {
+  var parsedUrl = url.parse(req.url, true);
+  var pathname = parsedUrl.pathname;
+  if (pathname === '/browserTestReports') {
+    var information = req.body.browserTestInfo;
+    var i = 0;
+    ++i;
```

Comment:
What are you trying to do here?

Reply:
I was trying to write the JSON recieved file by file at the location `coverage/lcov-vreport/browserData/${i}.txt` .

 I had seen an example like the following one `fs.writeFileSync(path.join(testSnapshotDir, (page + 1) + '.png'),
                         testSnapshot);` , but was confused on the file names so I derectly indexed them as of now. 

You could obviously suggest some better methods.